sos/plugins/gnocchi.py

```diff
@@ -10,44 +10,32 @@
 # See the LICENSE file in the source distribution for further information.
 
 import os
-from sos.plugins import Plugin, RedHatPlugin
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
 
 
-class GnocchiPlugin(Plugin, RedHatPlugin):
+class Gnocchi(Plugin):
     """Gnocchi - Metric as a service"""
     plugin_name = "gnocchi"
 
     profiles = ('openstack', 'openstack_controller')
 
-    packages = (
-        'openstack-gnocchi-metricd', 'openstack-gnocchi-common',
-        'openstack-gnocchi-statsd', 'openstack-gnocchi-api',
-        'openstack-gnocchi-carbonara'
-    )
-
     requires_root = False
-
     var_puppet_gen = "/var/lib/config-data/puppet-generated/gnocchi"
```

Comment:
If we're moving the `var_puppet_gen` file collections to the RedHat specific plugin, should we not move the definition of`self.var_puppet_gen` in the base `Gnocchi` class out as well?

Or conversely, should those file collections be updated to account for the apache package name differences as well?